app/src/main/sentences/en/joke.yml

```
@@ -0,0 +1,3 @@
+command:
+  - tell me a? another|good? joke
+  - i would like (to hear)? a? another|good? joke
```

Here are a few more alternatives
```suggestion
  - (tell me?)|(i (d|would like)|want (to hear)?)|say|(do you know|have)|got a|another|any? good? joke<s?>|pun<s?>|gag<s?>|prank<s?>
  - (make me laugh)|(entertain me)|(cheer me up) (with a|another|any? good? joke<s?>|pun<s?>|gag<s?>|prank<s?>)?
  - say something|anything fun|funny
```